app/templates/search/agenda-items.hbs

```diff
@@ -1,57 +1,79 @@
-            </option>
-          {{/each}}
-        </select>
-      </div>
-      <div class="auk-u-mb-2">
-        <Auk::Label>{{t "type"}}</Auk::Label>
-        <Auk::CheckboxList>
-          <AuControlCheckbox
-            @label={{t "note"}}
-            @checked={{this.includeNotas}}
-            @onChange={{this.toggleIncludeNotas}}
+{{page-title (t "agendaitems")}}
+<AuMainContainer as |m|>
+  <m.sidebar>
+    <div class="au-c-sidebar">
+      <div class="au-c-sidebar__content">
+        <Search::Navigation />
+        <div class="au-o-box au-o-box--small au-o-flow au-o-flow--tiny">
+          <Search::DateRangeFilter 
+            @onDateFromChange={{this.onDateFromChange}}
+            @onDateToChange={{this.onDateToChange}}
+            @onDateToClear={{set this "dateTo" undefined}}
+            @onDateFromClear={{set this "dateFrom" undefined}}
+            @dateFrom={{this.dateFrom}}
+            @dateTo={{this.dateTo}}
           />
-          <AuControlCheckbox
-            @label={{capitalize (t "announcement")}}
-            @checked={{this.includeMededelingen}}
-            @onChange={{this.toggleIncludeMededelingen}}
-          />
-        </Auk::CheckboxList>
-      </div>
-      <div>
-        <Auk::Label>{{t "version"}}</Auk::Label>
-        <AuControlCheckbox
-          @label={{t "only-on-final-agenda"}}
-          @checked={{this.latestOnly}}
-          @onChange={{this.toggleLatestOnly}}
-        />
+          <Search::MinisterFilter />
+          <div class="au-c-card au-o-box au-o-box--small">
+            <Auk::Label>{{t "type"}}</Auk::Label>
+            <Auk::Checkbox
+              @checked={{this.includeNotas}}
+              @label={{t "note"}}
+              {{on "input" this.toggleIncludeNotas}}
+            />
+            <Auk::Checkbox
+              @checked={{this.includeMededelingen}}
+              @label={{capitalize (t "announcement")}}
+              {{on "input" this.toggleIncludeMededelingen}}
+            />
+          </div>
+          <div class="au-c-card au-o-box au-o-box--small">
+            <Auk::Label>Agenda versie</Auk::Label>
+            <Auk::Checkbox
+              @checked={{this.latestOnly}}
+              @label={{t "only-on-final-agenda"}}
+              {{on "input" this.toggleLatestOnly}}
+            />
+          </div>
+        </div>
       </div>
     </div>
-  </div>
-  <div class="auk-o-grid-col-10">
-    <div>
+  </m.sidebar>
+  <m.content @scroll={{true}}>
+    <div class="au-o-box au-o-box--small">
       {{#if this.model.length}}
+        <div
+          class="au-u-flex au-u-flex--no-wrap au-u-flex--spaced au-u-flex--between au-u-flex--vertical-center au-u-margin-bottom-small"
+        >
+          <p class="au-u-muted">
+            <strong class="au-u-medium">1-{{this.size}}</strong>
+            resultaten voor
+            <strong class="au-u-medium">"{{this.searchTextBuffer}}"</strong>
+            in
+            <strong>agendapunten</strong>
```

Comment:
Strings should come from translations file. Either split up into parts or using one long strings with params (e.g. `  "block-user-are-you-sure": "Ben je zeker dat je <strong>{user}</strong> wil blokkeren uit Kaleidos?"`, which you then invoke like this `{{t "block-user-are-you-sure" user=@user.name}}`

Reply:
Same applies for the search/cases routes